src/component/TodoItem.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Todos from '../mock/data';
+
+class TodoItem extends Component {
```

Comment:
### Nitpick

Since you're not interacting with state here, can you kindly make use of a functional component instead?